code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -411,20 +411,27 @@ Congratulations! You are now trained for xenobiology research!"}
 	return 1
 
 /obj/item/weapon/abductor_baton/attack(mob/target, mob/living/user)
-	if(!IsAbductor(user))
+	if (!IsAbductor(user))
 		return
 
-	if(isrobot(target))
+	if (isrobot(target))
 		..()
 		return
 
-	if(!isliving(target))
+	if (!isliving(target))
```

Comment:
https://github.com/tgstation/-tg-station/blob/master/CONTRIBUTING.md#operators-and-spaces

Just for consistency.